dataverse-persistence/src/main/resources/Bundle_pl.properties

```diff
@@ -1283,6 +1283,7 @@ dataset.submit.failure=Nie uda\u0142o si\u0119 przekaza\u0107 zbioru danych - {0
 dataset.submit.failure.null=Nie mo\u017Cna przekaza\u0107 do weryfikacji. Zbi\u00F3r danych jest nullem.
 dataset.submit.failure.isReleased=Najnowsza wersja zbioru danych zosta\u0142a ju\u017C opublikowana. Do weryfikacji mo\u017Cna przekaza\u0107 tylko wersje robocze.
 dataset.submit.failure.inReview=Nie mo\u017Cna przekaza\u0107 do do weryfikacji tego zbioru danych poniewa\u017C jest on ju\u017C weryfikowany.
+dataset.submit.failure.noFiles=Ta wersja robocza zbioru danych nie zawiera żadnych plików i nie może zostać przesłana do weryfikacji. Prosimy dodać co najmniej jeden plik.
```

Comment:
Polish letters should be written with unicode

Reply:
Ahh I didn't had automatic Unicode translation setup at home in IJ :<.